3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/CampaignAttributionEnrichment.scala

```diff
@@ -73,7 +74,12 @@ object CampaignAttributionEnrichment extends ParseableEnrichment {
         content   <- ScalazJson4sUtils.extract[List[String]](config, "parameters", "fields", "mktContent")
         campaign  <- ScalazJson4sUtils.extract[List[String]](config, "parameters", "fields", "mktCampaign")
 
-        enrich =  CampaignAttributionEnrichment(medium, source, term, content, campaign)
+        customClickMap = ScalazJson4sUtils.extract[Map[String, String]](config, "parameters", "fields", "mktClickId").fold(
```

Comment:
Could you add a comment like // Assign empty Map on missing property for backwards compatibility with 1-0-0...